tests/offset_overflow_test.rs

```diff
@@ -0,0 +1,17 @@
+pub mod common;
+
+use common::table_field_generated::example as reader;
+use common::table_field_generated_verifier::example as verifier;
+use common::table_field_generated_verifier::get_root;
+
+#[test]
+fn test_soffset_overflow() {
```

Comment:
Only add a test for overflow when following soffset.

It is hard to test other overflows since they require a huge buffer.